src/test/integration/channel_metadata_for_bogus_channel_spec.js

```diff
@@ -1,12 +1,29 @@
 require('./integration_config.js');
 
 var channelName = "no_way_jose90928280xFF";
-var thisChannelResource = channelUrl + "/" + channelName;
-var testName = "channel_metadata_for_bogus_channel_spec";
+var channelResource = channelUrl + "/" + channelName;
 
-utils.configureFrisby();
+describe(__filename, function () {
 
-frisby.create(testName + ': Fetching channel metadata for nonexistent channel')
-    .post(thisChannelResource)
-    .expectStatus(404)
-    .toss();
+    it('gets channel metadata for a nonexistent channel', function (done) {
+        var url = channelResource;
+        var headers = {};
+        var body = '';
+
+        // TODO: i don't think POST is the correct verb for this test
+        // I'm keeping it as POST for now though since thats what the
+        // Frisby test did.
```

Comment:
I think this is supposed to test changing a non-existent channel?  Seems low value.